docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -219,7 +219,7 @@ Use this dashboard to:
     * View Max Success Server Latency.
     * View transactions by storage account.
 
-<img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Overview.png')} alt="Azure Storage Overview dashboard" style={{border: '1px solid gray'}} width="800" />
```

Comment:
add the enable defender for azure storage https://learn.microsoft.com/en-us/azure/defender-for-cloud/defender-for-storage-azure-portal-enablement?tabs=enable-storage-account